packages/neos-ui/src/Containers/RightSideBar/Inspector/index.js

```diff
@@ -250,6 +250,8 @@ export default class Inspector extends PureComponent {
                             $get('views', group).count() > 0
                         ), false))
 
+                        .sort((a, b) => $get('position', a) > $get('position', b))
```

Comment:
Maybe it makes sense to support positional array sorter here? I'm not sure if it was the case in the old UI though, but could be useful.

Reply:
It was not the case, but yes can be a nice feature. 

If we can proceed with the bugfix first. Then the feature ;)

Reply:
Ok!